### PR TITLE
docs: adapt example queries to chinook

### PR DESCRIPTION
**Pull request by aljazerzen**
Start on #1666

### CODE REVIEW

File: website/content/_index.md [hunk 1]
```diff
@@ -12,24 +12,28 @@ hero_section:
     link: https://prql-lang.org/book/
     label: "Reference"
   prql_example: |
-    from employees
-    filter start_date > @2021-01-01
```
Comment (max-sixty) [inline website/content/_index.md:16]: WDYT about keeping a date one? I think that's a nice PRQL feature

File: playground/src/examples.js [hunk 2]
```diff
@@ -1,69 +1,57 @@
...
-select [dept_name, title, avg_salary]
+# anti-join soundtracks
+join side:left s=soundtracks [==track_id]
+filter s.track_id == null
+
+# limit to kicker genres
+join g=high_energy [==genre_id]
+
+# format output
+select [t.track_id, track = t.name, genre = g.name]
```
Comment (max-sixty) [inline playground/src/examples.js:53]: Should our standard formatting have a space here? I generally find having no space keeps the lvalue & rvalue visually bound in a list of lots of items like this (minor point obv!)

**Review by max-sixty — APPROVED** [review]
Very nice! Thanks

File: website/content/_index.md (hunk 1 above)
Comment (aljazerzen) [reply website/content/_index.md:16]: Yup, good point.

File: playground/src/examples.js (hunk 2 above)
Comment (aljazerzen) [reply playground/src/examples.js:53]: We can have an example showing off different styles we allow. Sometimes it's useful to segment things you are doing into understandable chunks.

**Pull request merged**